<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>911</width>
    <height>767</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Recipe</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(236, 230, 217);</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20pt &quot;Carefree Cyrillic&quot;;</string>
   </property>
   <property name="text">
    <string>Name:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="name">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>391</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
font: 18pt &quot;Carefree Cyrillic&quot;;
border: none;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20pt &quot;Carefree Cyrillic&quot;;</string>
   </property>
   <property name="text">
    <string>Ingredients:</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="table">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>581</width>
     <height>281</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
font: 16pt &quot;Carefree Cyrillic&quot;;
border: none;</string>
   </property>
   <attribute name="horizontalHeaderCascadingSectionResizes">
    <bool>true</bool>
   </attribute>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>125</number>
   </attribute>
   <attribute name="horizontalHeaderMinimumSectionSize">
    <number>49</number>
   </attribute>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderShowSortIndicator" stdset="0">
    <bool>true</bool>
   </attribute>
   <column>
    <property name="text">
     <string>Ingredient</string>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="whatsThis">
     <string/>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Availability</string>
    </property>
    <property name="toolTip">
     <string/>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Count</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Units</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="refresh">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>290</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(254, 218, 113);
border: none;
color: rgb(95, 95, 95);
font: 10pt &quot;Arial&quot;;</string>
   </property>
   <property name="text">
    <string>Refresh</string>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="addIngr">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>360</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(254, 218, 113);
border: none;
color: rgb(95, 95, 95);
font: 10pt &quot;Arial&quot;;</string>
   </property>
   <property name="text">
    <string>Add new ingredient</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>430</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20pt &quot;Carefree Cyrillic&quot;;</string>
   </property>
   <property name="text">
    <string>Recipe:</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="recipe">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>470</y>
     <width>891</width>
     <height>231</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
font: 10pt &quot;Arial&quot;;
border: none;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="save">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>710</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(254, 218, 113);
border: none;
color: rgb(95, 95, 95);
font: 12pt &quot;Arial&quot;;</string>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QLabel" name="imageView">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>20</y>
     <width>251</width>
     <height>251</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="openImage">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>290</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(254, 218, 113);
border: none;
color: rgb(95, 95, 95);
font: 10pt &quot;Arial&quot;;</string>
   </property>
   <property name="text">
    <string>Open image</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
